src/routes/events/events.router.js

```diff
@@ -0,0 +1,52 @@
+const express = require('express');
+const { check } = require('express-validator');
+
+const events = require('./events.controller');
+const { validateBody } = require('../../middleware/validate-body');
+
+const router = express.Router();
+
+router.get('', events.get);
```

Comment:
Beautiful!

One other thing I forgot to mention is that you might consider putting something into the path part of the url routing (e.g. `router.get('/events', events.get)`). That way your URLs will be much more indicative of what they point to.  Using an empty path like this also breaks down as soon as you have more resources (e.g. users), so it's a good idea to design with that in mind.

Reply:
I think that this is covered since it's imported into the main router file at `api/index.js`  but this is a very good thing to keep in mind!

Reply:
oh! you're right, my mistake.  Good work!